web/client/codechecker_client/cli/store.py

```diff
@@ -257,6 +257,17 @@ def add_arguments_to_parser(parser):
                              "match will be removed. You may also use Unix "
                              "shell-like wildcards (e.g. '/*/jsmith/').")
 
+    parser.add_argument('--zip_loc',
```

Comment:
```suggestion
    parser.add_argument('--zip-loc',
```
Conventionally we use dash in flag names.